dotcom-rendering/src/web/components/FixedLargeSlowXIV.tsx

```diff
@@ -59,12 +56,7 @@ export const FixedLargeSlowXIV = ({
 						discussionId={primary.discussionId}
 					/>
 				</LI>
-				<LI
-					padSides={true}
-					showDivider={true}
-					showTopMarginWhenStacked={true}
```

Comment:
I've replaced top margin here with bottom padding on the element above. The reason for doing so is because it seemed as though padding on `UL`s can only be set for all breakpoints (no 'not on mobile' option). And once one element has padding on the bottom you need to set all of the spacing on the bottom of the elements, to avoid getting double-sized gaps.

So this seemed like the option which leaves shared code intact. But it also feels like `showTopMarginWhenStacked` would be the more _semantic_ way to express the desired gaps here. Might not be a big deal either way, but thoughts are very welcome! I'm thinking @oliverlloyd you might have some insight here?